### PR TITLE
feat: rest-fastify

### PR DESCRIPTION
**Pull request by Aperrix**
✅ Closes: #1395

### CODE REVIEW

**Comment by janpio** [comment]
Ah, the error message from CI is not super helpful here, let me explain:

```
executing .github/tests/./javascript/rest-fastify/run.sh (tests)
no test file set up for ./javascript/rest-fastify/package.json,
please create a test shell file at /home/runner/work/prisma-examples/prisma-examples/.github/tests/./javascript/rest-fastify/run.sh
```

This CI job executes a test shell script for each example (which is located in another folder, so people do not get it when they check out the example itself). Some examples:

https://github.com/prisma/prisma-examples/blob/master/.github/tests/javascript/rest-express/run.sh
https://github.com/prisma/prisma-examples/blob/master/.github/tests/javascript/graphql/run.sh
https://github.com/prisma/prisma-examples/blob/master/.github/tests/javascript/rest-nextjs/run.sh

You see they are pretty simple, and mostly just check if the server is responding with what we expect, so the project could successfully be installed, started and responds.

Let us know if you need any help to set that up for this project as well!

File: javascript/rest-fastify/src/index.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+        },
+      ],
+    },
+  });
+  res.send(draftPosts);
+});
+
+const server = app.listen(3000, () =>
+  console.log(
+    "🚀 Server ready at: http://localhost:3000\n⭐️ See sample requests: http://pris.ly/e/ts/rest-express#3-using-the-rest-api"
```
Comment (gregbenner) [inline javascript/rest-fastify/src/index.js:93]: *fastify ?

Comment (Aperrix) [reply javascript/rest-fastify/src/index.js]: My bad 😅

**Comment by Aperrix** [comment]
I hope this contribution will prove that between express and fastify there is no development cost but the perfs are incredibly better.

So please let express retire and be forgotten once and for all, thx 🙏

**Comment by nikolasburk** [comment]
Thanks a lot for this contributing this example @Aperrix, this was a huge help! FYI, we're using a "templating system" for our READMEs where we're composing the final example READMEs from smaller Markdown snippets (see this [example](https://github.com/prisma/prisma-examples/tree/master/.github/readmes/javascript/rest-express)). I'll still merge your PR and will adjust the README myself, just wanted to let you know in case you want to contribute further examples in the future 😄 🙏  Thanks again for taking the time to help!

**Pull request merged**